lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -355,6 +355,41 @@ e2function string entity:propPhysicalMaterial()
 	return ""
 end
 
+hook.Add( "CanDrive", "checkPropStaticE2", function( ply, ent ) return ent.propStaticE2 == nil end )
+e2function void entity:propStatic( number static )
+	if not PropCore.ValidAction( self, this, "static" ) then return end
+	if static ~= 0 and this.propStaticE2 == nil then
+		local phys = this:GetPhysicsObject()
+		this.propStaticE2 = phys:IsMotionEnabled()
```

Comment:
This variable is used for remembering the state (frozen or unfrozen) the entity was when calling propStatic(1), to possibly restore it later when calling propStatic(0). It is also used to disallow Drive'ing this entity.